### ISSUE TITLE
feature suggestion: detect actions and what module they come from

### ISSUE THREAD

**Issue by mgenev** [emberjs/ember-inspector#227]
It would be really helpful if among everything else ember inspector could detect the active actions and their origins.

I love your work btw :)

**Comment by kevinvangelder** [comment]
Yes, this feature would be great! I'm using a plugin as documented but it's not working. I can see the ember actions are bound to the element, `data-ember-action="1689"`, but I can't dig into it any deeper without the suggested tab in the Ember inspector.

**Comment by rwjblue** [comment]
Logging actions as they happen (where they were triggered and where they were handled) would definitely be neat.

**Comment by davidpatters0n** [comment]
:+1: for this

**Comment by jasonworden** [comment]
I'm interested in working on this. I'd also want it to work out-of-the-box (or maybe v2) with ember-concurrency tasks

**Comment by jasonworden** [comment]
I had in mind something like the [Redux dev tools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en) which has a history of all actions that go through the reducers

**Comment by rwwagner90** [comment]
@jasonworden we would definitely love to help you bring your visions to life 😄

**Comment by sandstrom** [comment]
I'm doing some issue gardening 🌱🌿 🌷 and came upon this issue. Since it's quite old I just wanted to ask if this is still relevant? If it isn't, maybe we can close this issue?

Actions are being replaced with vanilla functions, so this may not be relevant anymore?

By closing some old issues we reduce the list of open issues to a more manageable set.

**Comment by rwwagner90** [comment]
@sandstrom I think it's still relevant, but we need someone to want to take it on and work on it.

**Comment by sandstrom** [comment]
@rwwagner90 How about adding the 'Discussions' tab and move stuff like that there?

Realistically, many things that are suggested will never be added/built, and having a bunch of open issues makes it less obvious what the "real" issues are.

There is a handy button that can one-click move issues into discussions. One can also up-vote various posts there, to see e.g. interest for various ideas.

**Comment by jasonworden** [comment]
I no longer plan on taking this one up, but looking forward to it making its way into the tool someday!
